asciidoc/src/main/docs/asciidoc/basic/concepts.adoc

```diff
@@ -153,4 +153,4 @@ Bucket bucket = Bucket.builder()
     .withNanosecondPrecision()
     .withSynchronizationStrategy(SynchronizationStrategy.LOCK_FREE)
     .build()
-----
\ No newline at end of file
+----
```

Comment:
I don't know why I can't remove this change... never made this.
I've tried it already two times in 
[remove newline at end of file](https://github.com/bucket4j/bucket4j/pull/487/commits/be3614f7bf3dd699a76758923552c703f8b68cba) and [remove new line](https://github.com/bucket4j/bucket4j/pull/487/commits/408367a1921c58d5205cbf16340d4a70e57389bd)